### PR TITLE
Pycon US 2018

### LINKED ISSUE #494: PyCon US 2018
May 9-17 Cleveland

~~~ yaml
 - title: PyCon US 2018 
   dir: pycon-us-2018
   youtube_list: https://www.youtube.com/channel/UCsX05-2sVSH7Nx3zuk3NYuQ/videos
   language: en
   related_urls:
   - label: schedule
     url: https://us.pycon.org/2018/schedule/talks/
~~~

### PR DESCRIPTION
**Pull request by Daniel-at-github**
Fixes #494

### CODE REVIEW

**Comment by jonemo** [comment]
Out of curiosity: Did you get the Friday lightning talk info from a source or did you compile that manually? I noticed that it's the one round of lightning talks for which no timestamps are included in the Youtube description, yet it has the most details in this data :)

File: pycon-us-2018/videos/lightning-talks-friday-pycon-2018.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
...osengarten\n     - Music from chaos: Audiofying the Lorenz Attractor\n   * - 34:05_\n     - Sahuna Gordon-McKeon\n     - Legacies of disruption\n   * - 39:02_\n     - Travis Oliphant\n     - More sustainable Open Source!\n   * - 42:07_\n     - James Powell\n     - Code or die\n   * - 48:56_\n     - Scott Shawcroft\n     - Circuit Python\n   * - 51:57_\n     - Sean Harrington\n     - Hacking in 23&Me and DbSNP\n\n\n.. _00:40: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=00m40s\n.. _05:01: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=05m01s\n.. _10:05: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=10m05s\n.. _14:44: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=14m44s\n.. _18:07: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=18m07s\n.. _23:00: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=23m00s\n.. _28:50: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=28m50s\n.. _34:05: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=34m05s\n.. _39:02: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=39m02s\n.. _42:07: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=42m07s\n.. _48:56: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=48m56s\n.. _51:57: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=51m57s\n",
```
Comment (jonemo) [inline pycon-us-2018/videos/lightning-talks-friday-pycon-2018.json]: Mahmoud Hasemi's talk title is incorrect in the description here, it should be "Restructuring Data in Python"

Comment (jonemo) [reply pycon-us-2018/videos/lightning-talks-friday-pycon-2018.json]: And one more comment for this line, this one's just a typo: "Hacking in 23&Me and DbSNP" should be "Hacking on 23&Me and DbSNP"

File: pycon-us-2018/videos/lightning-talks-saturday-evening-pycon-2018.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "copyright_text": "Standard YouTube License",
+  "description": "",
```
Comment (jonemo) [inline pycon-us-2018/videos/lightning-talks-saturday-evening-pycon-2018.json]: I just compiled this while watching the video on Youtube:
```
.. list-table:: Lightning Talks
   :widths: 10 30 60
   :header-rows: 1
   * - Time
     - Speaker
     - Title
   * - 00:10_
     - Caleb Collins-Parks
     - AREPL - A real-time python evaluator
   * - 04:35_
     - Michael Foord
     - Abstractions
   * - 10:27_
     - Turicas (Álvaro) Justen
     - Roasting Coffee with Python
   * - 16:27_
     - Petr Viktorin
     - Blinky's Async Adventure 
   * - 22:17_
     - 
     - Type Hint 
   * - 26:25_
     - Scott Sanderson
     - A Class is a Poor Man's...Loop 
   * - 31:25_
     - Nick Humrich
     - RIP HTTP 
   * - 36:08_
     - Allen Downey
     - Modeling and Simulation in Python 
   * - 40:00_
     - Petr Mitev
     - Python in Arch 
   * - 44:53_
     - Steven C. Howell
     - Bayesian Hacking 
   * - 50:47_
     - James Bennett Saxon
     - Code Lives & Things can happen 
   * - 56:42_
     - Andrés García García
     - Why should we care about Processing 
   * - 1:01:40_
     - Meredydd Luff
     - Making the web more pythonic 
   * - 1:07:10_
     - Kenneth Reitz
     - Running for the board﻿

.. _00:10: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=00m10s
.. _04:35: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=04m35s
.. _10:27: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=10m27s
.. _16:27: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=16m27s
.. _22:17: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=22m17s
.. _26:25: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=26m25s
.. _31:25: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=31m25s
.. _36:08: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=36m08s
.. _40:00: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=40m00s
.. _44:53: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=44m53s
.. _50:47: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=50m47s
.. _56:42: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=56m42s
.. _1:01:40: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=61:40
.. _1:07:10: https://www.youtube.com/watch?v=c-I0md_3fbQ&t=67:10
```

**Comment by Daniel-at-github** [comment]
> Out of curiosity: Did you get the Friday lightning talk info from a source or did you compile that manually? 

@jonemo I did it manually and formated with 
~~~ bash
tools/csv2rst.py --json pycon-us-2018/videos/lightning-talks-friday-pycon-2018.json lightning2018.csv https://www.youtube.com/watch?v=bTAFl8P2DkE
~~~

`--json` simplifies the process substituting the description with the table.

File: pycon-us-2018/videos/lightning-talks-friday-pycon-2018.json (hunk 1 above)
Comment (Daniel-at-github) [reply pycon-us-2018/videos/lightning-talks-friday-pycon-2018.json]: Mahmoud Hasemi's talk title should be a "copy" and "paste" bug :sweat_smile:
"Hacking on 23&Me and DbSNP" is totally on me :innocent: 

(And I forgot to add the speakers)

File: pycon-us-2018/videos/lightning-talks-saturday-evening-pycon-2018.json (hunk 2 above)
Comment (Daniel-at-github) [reply pycon-us-2018/videos/lightning-talks-saturday-evening-pycon-2018.json]: Great! :+1: 

I will add the `related_urls` when I see this lightning.

This pull request have :ballot_box_with_check: Allow edits from maintainers. If you want, you can add more changes to this pull request. It should be something as adding my repo as remote, modifying it in `Daniel-at-github:pycon-us-2018` branch.

**Comment by redapple** [comment]
Thank you @Daniel-at-github and @jonemo !

**Pull request merged**